src/point_vector_fallbacks.jl

```diff
@@ -27,6 +27,9 @@ macro manifold_element_forwards(T, Twhere, field::Symbol)
             function ManifoldsBase.allocate(p::$T, ::Type{P}) where {P,$Twhere}
                 return $T(allocate(p.$field, P))
             end
+            function allocate(p::$T, ::Type{P}, dims::Tuple) where {P,$Twhere}
+                return $T(allocate(p.value, P, dims))
```

Comment:
```suggestion
                return $T(allocate(p.$field, P, dims))
```
It looks fine with this small change.

Reply:
Thanks for noticing, this is indeed a copy-paste-error. And I will check for a test to cover these as well, for sure.